synapse/handlers/federation_event.py

```diff
@@ -543,15 +543,15 @@ async def update_state_for_partial_state_event(
             )
             if context.partial_state:
                 # this can happen if some or all of the event's prev_events still have
-                # partial state - ie, an event has an earlier stream_ordering than one
-                # or more of its prev_events, so we de-partial-state it before its
-                # prev_events.
+                # partial state. We were careful to only pick events from the db without
+                # partial-state, so that implies that the prev event has been persisted
+                # (with partial state) since we did the query.
```

Comment:
nit: I think we're missing a word or two?
```suggestion
                # partial state. We were careful to only pick events from the db without
                # partial-state prev events, so that implies that a prev event has
                # been persisted (with partial state) since we did the query.
```

Reply:
thanks!